R/qza_to_tse.R

```diff
@@ -0,0 +1,108 @@
+#' Generates a TreeSummarizedExperiment (TSE) object from .qza artifacts
+#'\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'
+#' Construct a TreeSummarizedExperiment object from multiple 
+#' qiime2 artifacts (.qza). 
+#' '\code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#' Embedded metadata for provenance is not maintained in this function and 
+#' instead \code{read_qza()} should be used.
+#' 
+#' @param features file path for artifact containing a feature (OTU/SV) table
+#' @param tree file path for artifact containing a tree
+#' @param taxonomy file path for artifact containg taxonomy
+#' @param metadata file path for a qiime2-compliant TSV metadata file
+#' @param tmp a temporary directory that the object will be decompressed to.
+#' @return a TreeSummarizedExperiment object
+#' \code{\link[TreeSummarizedExperiment:SummarizedExperiment-class]{TreeSummarizedExperiment}}
+#' 
+#' @import TreeSummarizedExperiment
+#' @importFrom S4Vectors SimpleList DataFrame make_zero_col_DFrame
+#' @importFrom SummarizedExperiment colData colData<-
+#' 
+#' @examples 
+#' (Data is from tutorial
+#' \url{https://docs.qiime2.org/2020.2/tutorials/moving-pictures/)}
+#' 
+#' \donttest{tse <-qza_to_tse(
+#'     features="path_to_table.qza",
+#'     tree="path_to_rooted-tree.qza",
+#'     taxonomy="path_to_taxonomy.qza",
+#'     metadata = "path_to_sample-metadata.tsv"
+#' )}
+#' 
+#' @export
+#' 
+#' @author
+#' Leo Lahti
+#' Noah de Gunst
```

Comment:
Just put on one line: Leo Lahti and Noah de Gunst

Reply:
merged into one line